docker-for-mac/release-notes.md

```diff
@@ -13,13 +13,43 @@ This page contains information about the new features, improvements, known issue
 
 > **Important**
 >
-> Starting with Docker Desktop 3.0.0, Stable and Edge releases are combined into a single release stream for all users. Updates to Docker Desktop will now be available automatically as delta updates from the previous version. This means, when there is a newer version of Docker Desktop, it will be automatically downloaded to your machine. All you need to do is to click **Update and restart** from the Docker menu to install the latest update.
+> Starting with Docker Desktop 3.0.0, Stable and Edge releases are combined into a single release stream for all users. Updates to Docker Desktop will be available automatically as delta updates from the previous version. When an update is available, Docker Desktop displays an icon to indicate the availability of a newer version. Whenever convenient, you can start the download of the update in the background. When the download finishes, all you need to do is to click **Update and restart** to install the latest update.
 {: .important }
 
+## Docker Desktop 3.3.0
+2021-04-08
+
+> [Download](https://desktop.docker.com/mac/stable/amd64/Docker.dmg)
+
```

Comment:
We should have a **New** section here to talk about the new update mechanism. Here's the draft text. Let me know your thoughts @stephen-turner @dieucao 

You can now specify when to download and install a Docker Desktop update. When an update becomes available, Docker Desktop displays an icon to indicate the availability of a newer version. You can download the update in the background whenever convenient. When the download is complete, all you need to do is to click **Update and restart** to install the latest update.

Reply:
Thanks, @usha-mandya, LGTM.